File: csharp-dotnet2/src/main/CsharpDotNet2/estuary-client/Model/UtilContentAddResponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class UtilContentAddResponse {
    /// <summary>
    /// Gets or Sets Cid
    /// </summary>
    [DataMember(Name="cid", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "cid")]
    public string Cid { get; set; }

    /// <summary>
    /// Gets or Sets EstuaryId
    /// </summary>
    [DataMember(Name="estuaryId", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "estuaryId")]
    public int? EstuaryId { get; set; }

    /// <summary>
    /// Gets or Sets Providers
    /// </summary>
    [DataMember(Name="providers", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "providers")]
    public List<string> Providers { get; set; }

    /// <summary>
    /// Gets or Sets RetrievalUrl
    /// </summary>
    [DataMember(Name="retrieval_url", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "retrieval_url")]
    public string RetrievalUrl { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class UtilContentAddResponse {\n");
      sb.Append("  Cid: ").Append(Cid).Append("\n");
      sb.Append("  EstuaryId: ").Append(EstuaryId).Append("\n");
      sb.Append("  Providers: ").Append(Providers).Append("\n");
      sb.Append("  RetrievalUrl: ").Append(RetrievalUrl).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: dart/lib/model/util_content_add_response.dart
part of swagger.api;

class UtilContentAddResponse {
  
  String cid = null;

  int estuaryId = null;

  String estuaryRetrievalUrl = null;

  List<String> providers = [];

  String retrievalUrl = null;

  UtilContentAddResponse();

  @override
  String toString() {
    return 'UtilContentAddResponse[cid=$cid, estuaryId=$estuaryId, estuaryRetrievalUrl=$estuaryRetrievalUrl, providers=$providers, retrievalUrl=$retrievalUrl, ]';
  }

  UtilContentAddResponse.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    cid = json['cid'];
    estuaryId = json['estuaryId'];
    estuaryRetrievalUrl = json['estuary_retrieval_url'];
    providers = (json['providers'] as List).map((item) => item as String).toList();
    retrievalUrl = json['retrieval_url'];
  }

  Map<String, dynamic> toJson() {
    return {
      'cid': cid,
      'estuaryId': estuaryId,
      'estuary_retrieval_url': estuaryRetrievalUrl,
      'providers': providers,
      'retrieval_url': retrievalUrl
     };
  }

  static List<UtilContentAddResponse> listFromJson(List<dynamic> json) {
    return json == null ? new List<UtilContentAddResponse>() : json.map((value) => new UtilContentAddResponse.fromJson(value)).toList();
  }

  static Map<String, UtilContentAddResponse> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, UtilContentAddResponse>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new UtilContentAddResponse.fromJson(value));
    }
    return map;
  }
}
